scripts/powershell/Verify-And-Merge-PRs.ps1

```diff
@@ -0,0 +1,159 @@
+param(
```

Comment:
A nice to have as a follow-up: Some way to run this script locally to check the status of all the associated PRs.